src/AudioToolbox/MusicTrack.cs

```diff
@@ -129,6 +129,20 @@ internal override IntPtr ToUnmanaged ()
 
 	public class MusicEventUserData : MidiRawData {
 		public MusicEventUserData () {}
+
+		internal MusicEventUserData (IntPtr handle)
+		{
+			if (handle == IntPtr.Zero)
+				throw new ArgumentNullException (nameof(handle));
```

Comment:
minor, lack of space before `(`, but it means your IDE editor is not configured to follow Mono's guideline (and it should be to avoid those)